src/index.js

```diff
@@ -1,53 +1,16 @@
-import './style.css';
-
-const ToDo = [
-  {
-    description: 'Write Notes',
-    completed: 'false',
-    index: 0,
-  },
-  {
-    description: 'Early exercise',
-    completed: 'true',
-    index: 1,
-  },
-  {
-    description: 'Do laundry',
-    completed: 'true',
-    index: 2,
-  },
-  {
-    description: 'Buy some groceries',
-    completed: 'false',
-    index: 3,
-  },
-  {
-    description: 'Visit Grandma',
-    completed: 'false',
-    index: 4,
-  },
-];
-
-const rotate = document.querySelector('.refresh');
-rotate.addEventListener = ('click', () => {
-  rotate.classList.toggle('active');
-});
+/* eslint-disable max-classes-per-file */
 
-const todoTask = document.querySelector('.todo-tasks');
-
-const createElement = (task) => {
-  const elementDiv = document.createElement('div');
-  elementDiv.className = 'todo-lists the-elements';
-  const checkBox = document.createElement('input');
-  const todoParagraph = document.createElement('p');
-
-  checkBox.setAttribute('type', 'checkbox');
-  checkBox.checked = task.completed;
-  todoParagraph.innerHTML = task.description;
-  elementDiv.append(checkBox, todoParagraph);
-  todoTask.append(elementDiv);
-};
+import addTodo from './modules/AddTodo.js';
+import displayTodos from './modules/DisplayTodo.js';
+import { textField } from './modules/GetElements.js';
+import './style.css';
 
-ToDo.forEach((task) => {
-  createElement(task);
+document.addEventListener('DOMContentLoaded', () => {
+  textField.addEventListener('keydown', (e) => {
+    if (e.key === 'Enter') {
+      addTodo();
+      textField.value = '';
```

Comment:
- [ ] [OPTIONAL] You can currently add a new task by pressing the 'Enter' key on the keyboard. Know what would make your it even better, if users can add a task by pressing the button on the right as shown below 👇 
![image](https://user-images.githubusercontent.com/46839548/185754284-3774317e-8733-4815-b0bf-df017b70b7e4.png)